Signal/src/Models/TSMessageAdapaters/TSVideoAttachmentAdapter.m

```diff
@@ -121,23 +121,16 @@ - (UIView *)mediaView {
             _videoPlayButton.frame = CGRectMake((size.width / 2) - 18, (size.height / 2) - 18, 37, 37);
             [self.cachedImageView addSubview:_videoPlayButton];
             _videoPlayButton.hidden = YES;
-            _maskLayer              = [CALayer layer];
```

Comment:
There was pre-existing logic to show upload progress for videos only, but it was broken.  I've fixed it and generalized it to all media views.  When we support arbitrary attachments, we can use the AttachmentUploadView there too.